### ISSUE TITLE
`k8sExporter` does not work

### ISSUE THREAD

**Issue by debajyoti-truefoundry** [siglens/charts#17]
https://github.com/siglens/charts/blob/16e7d3fd82fab50912924e21d543c1a712235467/charts/siglens/values.yaml#L47

```
December 30 15:51:02.076
{"level":"error","time":"2024-12-30T10:21:02Z","caller":"/app/pkg/sinks/elasticsearch.go:144","message":"Indexing failed: Not Found"}
December 30 15:51:02.076
{"level":"error","time":"2024-12-30T10:21:02Z","caller":"/app/pkg/sinks/elasticsearch.go:144","message":"Indexing failed: Not Found"}
December 30 15:51:02.076
{"level":"error","time":"2024-12-30T10:21:02Z","caller":"/app/pkg/sinks/elasticsearch.go:144","message":"Indexing failed: Not Found"}
```

I am unsure how to create the `kubernetes-events` index from the Siglens UI.

Chart detail:
```yaml
repo_url: https://siglens.github.io/charts
chart: siglens
version: 0.1.3
```

siglens version: `1.0.7`

**Comment by nkunal** [comment]
you don;t need to create the `kubernetes-events` index, it is automatically created when the first log/event is sent to SigLens.
Do you see any errors on the siglens side ?
The 404 suggests that the URL might not be correct

**Comment by debajyoti-truefoundry** [comment]
https://github.com/siglens/charts/blob/16e7d3fd82fab50912924e21d543c1a712235467/charts/siglens/templates/events-exporter.yaml#L68

This points to the headless service, which is incorrect unless the pod name is also specified. We need to correct it and point it here:
https://github.com/siglens/charts/blob/16e7d3fd82fab50912924e21d543c1a712235467/charts/siglens/templates/service-ingest.yaml#L5

**Comment by debajyoti-truefoundry** [comment]
event collector config:
```yaml
logLevel: info
throttlePeriod: 30
logFormat: json
route:
  routes:
    - match:
        - receiver: "siglens"
receivers:
  - name: "siglens"
    elasticsearch:
      hosts:
        - http://siglens-ingest-svc:8081/elastic/
      index: kubernetes-events
      useEventID: false
```


```
$ curl http://siglens-ingest-svc:8081/elastic/
{"name":"siglens-0","cluster_name":"siglens","cluster_uuid":"011f9533-9e1f-4756-a367-6425d7416ff3","version":{"number":"7.9.3","build_flavor":"oss","build_type":"tar","build_date":"2021-10-07T21:56:19.031608185Z","build_hash":"83c34f456ae29d60e94d886e455e6a3409bba9ed","build_snapshot":false,"lucene_version":"8.9.0","minimum_wire_compatibility_version":"6.8.0","minimum_index_compatibility_version":"6.0.0-beta1"}}
```

I do not see any logs in the siglens container.

**Comment by AndrewHess** [comment]
This is fixed by [this PR](https://github.com/siglens/siglens/pull/2026). We just released siglens 1.0.9 which has this fix, so if you use the new siglens version you should see the exporter working properly
